include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -272,4 +285,99 @@ inline typename seq_type::iterator erase_gap(seq_type & seq, typename seq_type::
 }
 //!\}
 
+namespace detail
+{
+
+/*!
+ * \brief               Create the formatted alignment output and add it to the provided debug_stream.
+ * \tparam alignment_t  The type of the alignment, must satisfy tuple_like_concept.
+ * \tparam idx          An index sequence.
+ * \param[in] stream    The output stream that receives the formatted alignment.
+ * \param[in] align     The alignment that shall be streamed.
+ */
+template<tuple_like_concept alignment_t, size_t ...idx>
+void stream_alignment(debug_stream_type & stream, alignment_t const & align, std::index_sequence<idx...> const & /**/)
+{
+    size_t const alignment_length = std::get<0>(align).size();
+    char const * indent = "        ";
+
+    // split alignment into blocks of length 50 and loop over parts
+    for (size_t used_length = 0; used_length < alignment_length; used_length += 50)
+    {
+        // write header
+        if (used_length != 0)
+            stream << std::endl;
+
+        stream << std::setw(7) << used_length << ' ';
+        for (size_t col = 1; col <= 50 && col + used_length <= alignment_length; ++col)
+        {
+            if (col % 10 == 0)
+                stream << ':';
+            else if (col % 5 == 0)
+                stream << '.';
+            else
+                stream << ' ';
+        }
+
+        // write sequences
+        stream << std::endl << indent;
```

Comment:
why std::endl and not '\n' and why not using setw and then align right: see https://en.cppreference.com/w/cpp/io/manip for available manipulators.